.travis.yml

```diff
@@ -0,0 +1,59 @@
+language: generic
+
+sudo: required
+
+services:
+- docker
+
+env:
+  matrix:
+    - MOCK_CONFIG=fedora-rawhide-x86_64
+
+script:
+  - PKG=$(git log -1 --pretty=%s | sed 's/:.*//')
+  - if [[ ! -d ${PKG} ]]; then echo "Directory ${PKG} not found"; exit 1; fi
+  - if [[ ! -f ${PKG}/${PKG}.spec ]]; then echo "No spec file (${PKG}.spec found in directory ${PKG}"; exit 1; fi
+  - |
+    {
+      H="/travis/${PKG}"
+      echo "#!/bin/bash -xe"
+      echo "dnf -q -y install fedora-review"
+      echo "rpmbuild -D'_sourcedir ${H}' -D'_srcrpmdir ${H}' -bs ${H}/${PKG}.spec"
+      echo "useradd -d ${H} -g mock review"
+      echo "chown -R review:mock ${H}"
+      echo "/travis/spinner.sh \"su - -c 'fedora-review -v --mock-config ${MOCK_CONFIG} -n ${PKG}' review\" || { cat ${H}/.cache/fedora-review.log; find  ${H}/review-${PKG} -name '*.log' -print -exec cat {} \\;; exit 1; }"
+      echo "find ${H}/review-${PKG} -name '*.log' -print -exec cat {} \\;"
+      echo "cat ${H}/review-${PKG}/licensecheck.txt"
+      echo "cat ${H}/review-${PKG}/review.txt"
+    } > review.sh
+  - chmod +x review.sh
+  - docker run --privileged -v "${PWD}:/travis:rw" -it fedora:latest /travis/review.sh
+
+after_success:
```

Comment:
You can basically ignore that whole block for now. In my setup at https://github.com/junghans/fedora-review, Travis will automatically upload the spec file,`review.txt` and the `src.rpm` to github.io, so one can directly paste it into a review bug.

We could do the same here, but that would need some more Travis setup, namely encrypting the private key of a deploy key with the Travis public key (Travis CLI: `travis encrypt-file id_rsa.private`).